lib/apis/v2/bgpconfig.go

```diff
@@ -25,6 +25,8 @@ const (
 	KindBGPConfigurationList = "BGPConfigurationList"
 )
 
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
```

Comment:
I think we should have newlines between the generator tags and our other comments, throughout this PR.

Reply:
yep, agreed. putting together a PR right now.

Reply:
note that I squashed the commits, its newlines + stripped down register.go